autotest.pm

```diff
@@ -126,6 +126,7 @@ sub loadtest ($script, %args) {
             sys.path.append("$inc")
             sys.path.append("$script_dir")
             import $name
+            print("Using python version: %s" % sys.version)
```

Comment:
This would print the Python version before each module. It would likely be better to print it just once. And why is this needed anyways?